templates/named.conf.erb

```diff
@@ -0,0 +1,29 @@
+options {
+          listen-on port 53 { any; };
+          directory       "/var/named";
+          dump-file       "/var/named/data/cache_dump.db";
+          statistics-file "/var/named/data/named_stats.txt";
+          memstatistics-file "/var/named/data/named_mem_stats.txt";
+          allow-query     { any; };
+          recursion yes;
+
+
+          dnssec-enable no;
+          dnssec-validation no;
+          dnssec-lookaside auto;
+
+          forward first;
+          forwarders {
+              <%= external_dns %>;
```

Comment:
Yeah I will need to implement that. Their is a include in the bash script, so it will fail if the file doesn't exist